src/main.ts

```diff
@@ -45,9 +47,15 @@ const run = async (): Promise<void> => {
 
     await setSummary(summary);
     await publishComment(token, serverUrl, title, comment, postNewComment);
+
+    // Create status checks if enabled
```

Comment:
The code below is self-explanatory, so no need for this comment. Alternatively if you feel that it need some explanation - you can move it out to a separate function